### PR TITLE
Ignore cache if AWS_NOCACHE is set.

### PR DESCRIPTION
**Pull request by mclarkson**
So I can force a new token (request for MFA) and I will know it will last for token-timeout!

### CODE REVIEW

File: awsprofile/__init__.py [hunk 1]
```diff
@@ -50,12 +51,12 @@ def parse_args(argv=sys.argv):
...
     command = argv[1:]
-    return (profile, command)
+    return (profile, command, cache)
 
 def main():
-    profile, command = parse_args()
+    profile, command, cache = parse_args()
     session = botocore.session.Session(profile=profile)
-    configure_cache(session)
+    if not cache: configure_cache(session)
```
Comment (jrstarke) [inline awsprofile/__init__.py]: The choice of variable name seems confusing here, as it seems to imply the opposite.

**Comment by mclarkson** [comment]
Now it's a double negative, 'if not nocache ...', but better than before.

**Comment by strongoose** [comment]
imho it might be better to have the if statement explicitly check `if nocache == 'true':`.
This avoids the slightly counterintuitive situation where `AWS_NOCACHE=true` and `AWS_NOCACHE=false` are actually identical: both will disable caching. It might also make the if statement slightly clearer as a bonus.

**Comment by mclarkson** [comment]
@ouroboros8 I agree, thanks! I have done as you suggested .. I should not have been so lazy.

**Comment by jrstarke** [comment]
Nice work, this should now be available: https://pypi.python.org/pypi/aws-profile/0.3.0

**Comment by mclarkson** [comment]
That's great, thanks!

**Pull request merged**